packages/backend/src/modules/tracked-txs/modules/l2-costs/L2CostsUpdater.test.ts

```diff
@@ -0,0 +1,260 @@
+import { Logger } from '@l2beat/backend-tools'
+import { EthereumAddress, ProjectId, UnixTime } from '@l2beat/shared-pure'
+import { expect, mockFn, mockObject } from 'earl'
+import { Knex } from 'knex'
+
+import { ViemRpcClient } from '../../../../peripherals/viem-rpc-client/ViemRpcClient'
+import { TrackedTxResult } from '../../types/model'
+import { TrackedTxId } from '../../types/TrackedTxId'
+import { TrackedTxConfigEntry } from '../../types/TrackedTxsConfig'
+import { L2CostsUpdater } from './L2CostsUpdater'
+import {
+  L2CostsRecord,
+  L2CostsRepository,
+} from './repositories/L2CostsRepository'
+
+const MIN_TIMESTAMP = UnixTime.now()
+
+describe(L2CostsUpdater.name, () => {
```

Comment:
no `update` test - we should test what it does in case of empty array, if array is not empty it should call the other method in the class and call l2CostRepo.addMany with the results

Reply:
[a8f22be](https://github.com/l2beat/l2beat/pull/3007/commits/a8f22be321c91c6d012343d849d3bb9306dc0ffa)